app/models/user.rb

```diff
@@ -38,4 +38,8 @@ def my_issues
   def my_pivotal_stories
     pivotal_stories
   end
+
+  def owned_pulls
+    PullRequest.select{ |pr| pr.author.downcase == self.name.downcase}
```

Comment:
Maybe we can translate this into an association? I think now it's fetching all the PRs from the database to compare.